distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/cli/InstallPluginAction.java

```diff
@@ -225,7 +225,7 @@ public void execute(List<PluginDescriptor> plugins) throws Exception {
             terminal.println(logPrefix + "Installing " + pluginId);
             try {
                 if ("x-pack".equals(pluginId)) {
-                    handleInstallXPack(buildFlavor());
+                    throw new UserException(ExitCodes.CONFIG, "this distribution of Elasticsearch contains X-Pack by default");
```

Comment:
Do we even need to handle this anymore at this point. The "x-pack" plugin hasn't been a thing for many versions. I assume this was only added to ease migration after x-pack was folded into ES.

Reply:
That is a fair point, but I would like to think about that separately. We certainly might get users who have used ES before trying to install x-pack, though at some point we should definitely remove this.